services/QuillLMS/lib/tasks/vitally.rake

```diff
@@ -19,7 +19,12 @@ namespace :vitally do
     # has a batch limit of 100.
     CSV.parse(pipe_data, headers: true).each_slice(33) do |batch|
       batch_payload = batch.map do |row|
-        user = User.includes(:schools_users).find(row['externalId'])
+        # TIL `includes` makes a separate query to the db, and `joins` doesn't store the joined value in ActiveRecord, so if you want a single query to pull multiple models you need to use both?
```

Comment:
I usually have to look this up.  Reminds me of the other [options](https://www.bigbinary.com/blog/preload-vs-eager-load-vs-joins-vs-includes) preload and eager_load too.  I feel like at one point Rails changed how includes worked which I think is why I find this confusing.